molo/core/api/serializers.py

```diff
@@ -0,0 +1,32 @@
+from collections import OrderedDict
+
+from rest_framework import serializers
+
+from molo.core.api.compat import PageSerializer
+
+
+class PageChildrenField(serializers.Field):
+    """
+    Serializes the "children" field.
+
+    Example:
+    "children": {
+        "count": 1,
+        "listing_url": "/api/v1/pages/?child_of=2"
+    }
+    """
+    def get_attribute(self, instance):
+        return instance
+
+    def to_representation(self, page):
+        if page.get_children():
+            items = []
+            for item in page.get_children():
```

Comment:
Use `.iterator()` to be more [memory efficient](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.iterator).